### DISCUSSION TITLE
cannot find Kiosk mode

### DISCUSSION THREAD

**Discussion by Linde2023** [tijsverkoyen/HomeAssistant-FusionSolar#191] [General]
Could you please send me the steps I need to follow? When I log in to FusionSolar, I cannot find the Kiosk mode. When I go to the dashboard, I can see basic information, but I am unsure what to do next. (i dont see kiosk mode on the right corner side)
![fusion solar](https://github.com/user-attachments/assets/7ab5020b-473e-4eec-a197-51b23451859f)

**Comment by tijsverkoyen** [comment]
Click "Monitoring" at the top. And the button "Kiosk" will be on the right.

**Reply by tijsverkoyen** [reply]
@Linde2023 did that help you?

**Comment by Leshub** [comment]
Hello, sorry to hijack on open conversation but I am also not able to see the "kiosk" mode on the monitoring section.  Using Mac and tried Firefox and Safari.  Is there a way to derive the kiosk url from the regular one ?  Thank you.
![Capture d’écran 2025-04-25 à 11 41 01](https://github.com/user-attachments/assets/933b8c98-3448-4be5-b2c3-e14fda3e3865)

**Comment by tijsverkoyen** [comment]
The buttons should be right to "Try New Version". 
Can you click on the home icon.

**Comment by Leshub** [comment]
﻿Hi Tijs,Thanks for the quick reply.  I have a dropdown with 3 choices of view under Home but no view provides a kiosk option in the top right corner.  Also tried on Edge under window with same results.Thanks for your support.JPLe 27 avr. 2025 à 17:35, Tijs Verkoyen ***@***.***> a écrit :﻿
The buttons should be right to "Try New Version".
Can you click on the home icon.

**Comment by tijsverkoyen** [comment]
In that case I would contact your installer or Huawei support

**Comment by Leshub** [comment]
Will do, thanks.JPLe 28 avr. 2025 à 07:26, Tijs Verkoyen ***@***.***> a écrit :﻿
In that case I would contact your installer or Huawei support
